dnsmenace.py

```
@@ -1144,6 +1469,572 @@ async def bulk_lookup():
+
+    Discovers subdomains by testing common names against DNS servers.
+    Can query from specific countries to find geo-restricted subdomains.
+
+    Example:
+        dnsmenace brute example.com
+        dnsmenace brute example.com --doh
+        dnsmenace brute example.com -c CN -w custom_wordlist.txt
+    """
+    print_banner()
+
+    console.print(f"\n[bold cyan]Subdomain Enumeration[/bold cyan]")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)